### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by willsmith9182** [kovetskiy/mark#43]
I tried to upgrade to 3.4 (windows_x64 exe) when trying to get around #42  and I hit another issue:

```mark.exe -f .\things.md -u XXXX -p XXXX -b https://atom.atlassian.net/wiki/```

(same command works with 3.3.1 and produces #42)

.md file (same as other issue)
```
<!-- Space: DD -->
<!-- Parent: On-boarding docs -->
<!-- Parent: Systems Development -->
<!-- Parent: Application -->
<!-- Parent: BuildTools -->
<!-- Title: Repositories using BuildTools -->

<!-- Macro: :autogen:
     Template: ac:status
     Title: AUTOMATED
     Color: Yellow -->

:autogen:
this document is generated, do not edit manually.

----

## Build Tools Type: Cake

blah blah blha cake is good
blah blah

Here's some useful links:
 - [How does Cake BuildTools work](www.google.com)

### Repositories:

#### [NotMyRepo](https://github.com/NotMyOrg/OrMyRepo)
```Some Repository Desc from github api```

```
(the google link is temporary, but i'm tempted to leave it as a troll, maybe i should use lmjgtfy.com)

trace log:
```
2020-12-02 18:22:14 [TRACE]: loaded macro "@\\{([^}]+)\\}"
                             ├─ template: ac:link:user
                             ├─ expr: @\{([^}]+)\}
                             └─ config:
                                     Name: ${1}
2020-12-02 18:22:14 [TRACE]: loaded macro ":autogen:\r"
                             ├─ template: ac:status
                             ├─ expr: :autogen:
                             └─ config:
                                     Title: AUTOMATED
                                     Color: Yellow
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x20 pc=0x7c91ad]

goroutine 1 [running]:
github.com/kovetskiy/mark/pkg/mark.ResolveRelativeLinks(0xc00021ff60, 0xc00017c400, 0xb78, 0xc00, 0x892214, 0x1, 0xc00, 0x0, 0x0, 0xc00, ...)
        /home/runner/work/mark/mark/pkg/mark/link.go:33 +0x9d
main.main()
        /home/runner/work/mark/mark/main.go:216 +0x9e2

```

any pointers (pun not intended) 😄

**Comment by Bachib** [comment]
I am having the same issue

**Comment by kovetskiy** [comment]
Hey, I'm looking into that

**Comment by kovetskiy** [comment]
Hey, just fixed the issue, please check v3.5

The issue was introduced in #33

**Comment by willsmith9182** [comment]
Yup that's workin 👍
